docs/source/onnx.rst

```diff
@@ -872,6 +872,42 @@ The interface for specifying operator definitions is experimental;
 adventurous users should note that the APIs will probably
 change in a future interface.
 
+Autograd Function
+~~~~~~~~~~~~~~~~~
+
+Autograd functions can be used to compute operation results and gradients and save the history.
+More information on extending the torch.autograd engine can be found on this `page <https://pytorch.org/docs/stable/autograd.html#torch.autograd.Function>`_.
+To export a subclass of the torch.autograd.function, you can add a symbolic static method
+to your function class. The symbolic method should contain a set of ONNX operators that represent
+operator's behavior in ONNX. Here's an example for adding symbolic to the your autograd function: ::
+
+
+    class MyRelu(torch.autograd.Function):
+        @staticmethod
+        def forward(ctx, input):
+            ctx.save_for_backward(input)
+            return input.clamp(min=0)
+
+        @staticmethod
+        def backward(ctx, grad_output):
+            input, = ctx.saved_tensors
+            grad_input = grad_output.clone()
+            grad_input[input < 0] = 0
+            return grad_input
+
+        @staticmethod
+        def symbolic(g, self):
+            return g.op("Clip", self, g.op('Constant', value_t=torch.tensor(0, dtype=torch.float)))
+
+    class MyModel(torch.nn.Module):
+        def forward(self, x):
+            hard_swish = MyRelu.apply
+            return hard_swish(x)
+
+    input_tensor = torch.randn(2, 3, 224, 224, requires_grad=True)
+    torch.onnx.export(MyModel(), input_tensor, 'test.onnx', opset_version=12)
+
+
```

Comment:
If it's ok with you, I can add the new alternative way of export in this PR.

Reply:
Sure.